src/Components/ViewProject/ViewProject.tsx

```diff
@@ -1,7 +1,75 @@
-import React from 'react'
+import React, { useContext, useEffect, useState } from 'react'
+import avatar from '../../assets/avatar.png'
+import axios from 'axios'
+import { AuthContext } from '../../Context/AuthContext'
+import { useParams } from 'react-router-dom'
 
 export default function ViewProject() {
-  return (
-    <div>ViewProject</div>
+  const{id}=useParams()
+  const [viewPro,setViewPro]=useState([])
+const {baseUrl,reqHeaders}:any=useContext(AuthContext)
+useEffect(()=>{
+  axios.get(`${baseUrl}/Project/${id}`,{headers:reqHeaders}).then((response)=>{
+    console.log(response);
+    setViewPro(response?.data?.data)
+    
+  }).catch((error)=>{
+    console.log(error);
+    
+  })
+},[])
+  return (<>
+
+
+
+<div className="container">
+    
+    <div className="col-md-12">
+      <h3>Project Board</h3>
+    </div>
+   
+    <div className="col-md-8 m-auto ">
+      <div className=" col-md-8  m-auto content  bg-view rounded-3 text-center vh-50 mt-5">
+            <div className='pt-5 mb-5 mt-5'>
+      
+        </div>
+        <div className="secondary mb-3 pb-3 view">
+          {viewPro?.map((project)=><>
```

Comment:
Here the viewPro is not an array to loop through!! 
It's an object so it will contain a some fields, and you should display these fields